test/cli/esm-interop.spec.js

```diff
@@ -162,8 +162,8 @@ const fixture = [
     knexArgs: ['migrate:latest', isNode10 && `--esm`],
     expectedErrorMessage: isNode10
       ? 'Error: Cannot load module from .mjs'
-      : semver.gte(process.version, 'v14.13.0')
-      ? 'Unexpected export statement in CJS module'
+      : semver.major(process.version) === 14
```

Comment:
Are you sure about this change? This behaviour is different across specific 14.x versions, see https://github.com/knex/knex/pull/4055/files, so I would expect that less granular condition is going to break older versions.

Reply:
So I assume that correct fix would separate between pre-14.13.0, 14.13.0 and post-14.13.0

Reply:
That all depends - are these tests meant to be run on arbitrary minor versions, or are we talking the _latest_ minor version.  The CI is installing 14:
![image](https://user-images.githubusercontent.com/871639/97082201-dc316300-15ff-11eb-9aeb-2dbccd3cf97c.png)

I reckoned that the intention is to always test on the latest minor/patch.  But y, if it is important for the test framework to run on 14.13.0, then an additional specific check would be required.

Reply:
Considering that this is a test and adding additional check is not affecting runtime, I would prefer to be flexible with version support here.

Reply:
Tweaked.